installer/unix/install-cli

```diff
@@ -19,10 +19,30 @@ DEST="$DEST_PATH/$BINARY"
 MANIFEST_URL="https://binaries.particle.io/cli/${CHANNEL}/manifest.json"
 SHELL_CONFIG="$HOME/.bash_profile"
```

Comment:
testing over on my admittedly wonky ubuntu box, i found that `.bashrc` worked but `~/.bash_profile` did not... is that just down to my weird, hacked up machine?

Reply:
This would be a good question to ask the team. I'm not confident in my own knowledge of the difference between `.bashrc` and `.bash_profile` and what happens when only one of the files is present.

Reply:
seems ok for now 👉 https://s.slack.com/archives/C6SSY94UA/p1610586075001100

i'll monitor that thread and open a PR if tweaks are needed 👍